### PR TITLE
[FAB-17515] Support configuring BlockValidation policy for orderer group

### PR DESCRIPTION
**Pull request by caod123**
#### Type of change
- Bug fix

#### Description

Before we used to hardcode the BlockValidation policy to an ImplicitMetaAnyPolicy and we also did not enforce that the BlockValidation policy existed

#### Related issues

[FAB-17515](https://jira.hyperledger.org/browse/FAB-17515)

### CODE REVIEW

**Review by sykesm — APPROVED** [review]

File: internal/configtxgen/encoder/encoder_test.go [hunk 1]
```diff
@@ -46,7 +46,115 @@ func CreateStandardPolicies() map[string]*genesisconfig.Policy {
...
+			Type: "ImplicitMeta",
+			Rule: "ANY Writers",
+		},
+		"BlockValidation": {
+			Type: "ImplicitMeta",
+			Rule: "ANY Admins",
+		},
+	}
+}
+
```
Comment (sykesm) [inline internal/configtxgen/encoder/encoder_test.go]: Is there a reason we didn't call CreateStandardPolicies and then append a `BlockValidation` policy?

Comment (caod123) [reply internal/configtxgen/encoder/encoder_test.go]: oversight, will update 👍

File: internal/configtxgen/encoder/encoder_test.go [hunk 2]
```diff
@@ -46,7 +46,115 @@ func CreateStandardPolicies() map[string]*genesisconfig.Policy {
...
+			expectedImplicitMetaPolicy = &cb.ImplicitMetaPolicy{
+				SubPolicy: "Admins",
+				Rule:      cb.ImplicitMetaPolicy_ANY,
+			}
+			value, err = proto.Marshal(expectedImplicitMetaPolicy)
+			Expect(err).NotTo(HaveOccurred())
+			expectedBlockValidationPolicy := &cb.Policy{
+				Type:  int32(cb.Policy_IMPLICIT_META),
+				Value: value,
+			}
```
Comment (sykesm) [inline internal/configtxgen/encoder/encoder_test.go]: This looks like a good candidate for "marshal or panic." 

```go
Expect(cg.Policies["BlockValidation"].Policy).To(Equal(&cb.Policy{
    Type: int32(cb.Policy_IMPLICIT_META),
    Value: protoutil.MarshalOrPanic(&cb.ImplicitMetaPolicy{
        SubPolicy: "Admins",
        Rule: cb.ImplicitMetaPolicy_ANY,
     }),
})
```
I also think you can use probably build around CreateStandardPolicies for the first three policies.

**Review by sykesm — CHANGES_REQUESTED** [review]
I think this can be simplified a bit by delegating to the original `CreateStandardOrdererPolicies`. That would decrease the scope of this.

**Review by sykesm — APPROVED** [review]

**Comment by caod123** [comment]
@Mergifyio backport release-2.0

**Comment by caod123** [comment]
@denyeart @sykesm @jyellick can one of you convince Mergifyio to backport to release-2.0?

**Comment by lindluni** [comment]
DENIED!

**Comment by lindluni** [comment]
@Mergifyio backport release-2.0

**Pull request merged**